application/src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -1705,8 +1705,8 @@ type QueryType {
     preferences: PlanPreferencesInput,
     """
     Duration of the search window. This either starts at the defined earliest departure
-    time or ends at the latest arrival time. If this is not provided, a reasonable
-    search window is automatically generated. When searching for earlier or later itineraries
+    time or ends at the latest arrival time. If this is not provided or the value is set as null,
```

Comment:
Isen't `not provided` the same as `set to null` ?

Reply:
When it's not set, it uses the default value. When it's set as null, it doesn't.